### PR TITLE
Update header design when user is premium 

### PR DESCRIPTION
**Pull request by maxxcrawford**
Blocks #991

## Screenshots: 

Desktop:
image

Mobile: 
image

### CODE REVIEW

File: privaterelay/templates/base.html [hunk 1]
```diff
@@ -28,7 +28,7 @@
     <meta name="twitter:description" content="{% ftlmsg 'meta-description' %}">
     <meta name="twitter:image" content="{{request.scheme}}://{{request.META.HTTP_HOST}}{% static 'images/share-relay.jpg' %}">
   </head>
-  <body data-fxa-settings-url="{{ settings.FXA_SETTINGS_URL }}" data-site-origin="{{ settings.SITE_ORIGIN }}" data-google-analytics-id="{{ settings.GOOGLE_ANALYTICS_ID }}" data-debug="{{ settings.DEBUG }}">
+  <body class="{% with request.user.profile_set.first as user_profile %}{% if user_profile.has_unlimited %}is-premium{% endif %}{% endwith %}" data-fxa-settings-url="{{ settings.FXA_SETTINGS_URL }}" data-site-origin="{{ settings.SITE_ORIGIN }}" data-google-analytics-id="{{ settings.GOOGLE_ANALYTICS_ID }}" data-debug="{{ settings.DEBUG }}">
```
Comment (maxxcrawford) [inline privaterelay/templates/base.html:31]: This is the secret sauce for this feature – a logic switch to place a `.is-premium` class in the body.

File: privaterelay/templates/includes/header.html [hunk 2]
```diff
@@ -68,20 +67,20 @@
...
-    <nav class="flx flx-row ff-Met al-cntr header-nav hide-mobile">
+    <nav class="header-nav">
       <a class="{% if request.path == '/' or request.path == '/accounts/profile/' %} active-header-link {% endif %} header-link" href="/">{% ftlmsg 'nav-home' %}</a>
       <a class="{% if request.path == '/faq' %} active-header-link {% endif %} header-link" href="/faq">{% ftlmsg 'nav-faq' %}</a>
     </nav>
     {% include "includes/login.html" %}
   </div>
-  <nav class="flx flx-col ff-Met al-cntr mobile-menu show-mobile">
-    <button class="mobile-menu-toggle full-width ff-Met" aria-label="{% ftlmsg 'label-open-menu' %}"><span class="mobile-menu-label">{% ftlmsg 'nav-menu' %}</span></button>
-    <div class="mobile-menu-links flx flx-col">
```
Comment (maxxcrawford) [inline privaterelay/templates/includes/header.html:84]: 🧹  Cleaning up the utility classes.

File: static/scss/partials/banners.scss [hunk 3]
```diff
@@ -0,0 +1,89 @@
+.recruitment-banner, .micro-survey-banner {
+  text-align: center;
+  position: relative;
+  padding: 1em;
```
Comment (maxxcrawford) [inline static/scss/partials/banners.scss:4]: TODO: Migrate to Protocol tokens

File: static/scss/partials/header.scss [hunk 4]
```diff
@@ -0,0 +1,374 @@
...
+    margin: auto 0 auto auto;
+    font-family: $font-metropolis;
+    align-items: center;
+    @media screen and (max-width: 550px) {
+        display: none;
+    }
+}
+
+.sign-in {
+    padding: .75rem 1.25rem;
```
Comment (maxxcrawford) [inline static/scss/partials/header.scss:34]: Use protocol tokens

**Review by maxxcrawford — COMMENTED** [review]
- [ ] Migrate each file to use protocol tokens

**Review by codemist — APPROVED** [review]
Everything looks good! 👍

**Pull request merged**